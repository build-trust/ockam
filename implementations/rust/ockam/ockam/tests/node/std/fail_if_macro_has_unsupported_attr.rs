#[ockam::node(timeout = 100)]
async fn main(c: ockam::Context) {
    c.stop().await.unwrap();
}
