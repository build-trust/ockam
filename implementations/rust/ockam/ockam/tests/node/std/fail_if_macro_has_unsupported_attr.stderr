error: unknown attribute `timeout`
 --> tests/node/std/fail_if_macro_has_unsupported_attr.rs:1:15
  |
1 | #[ockam::node(timeout = 100)]
  |               ^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/node/std/fail_if_macro_has_unsupported_attr.rs:4:2
  |
4 | }
  |  ^ consider adding a `main` function to `$DIR/tests/node/std/fail_if_macro_has_unsupported_attr.rs`
